framework/include/mesh/ElemInfo.h

```diff
@@ -34,6 +34,16 @@ class ElemInfo
   const Elem * elem() const { return _elem; }
   Real volume() const { return _volume; }
   const Point & centroid() const { return _centroid; }
+  Real coordFactor() const { return _coord_transform_factor; }
+  Real & coordFactor() { return _coord_transform_factor; }
+  const std::vector<std::vector<dof_id_type>> & dofID() const { return _dof_ids; }
+  std::vector<std::vector<dof_id_type>> & dofID() { return _dof_ids; }
+
+  void setDofIDs(const std::vector<std::vector<dof_id_type>> & dof_ids)
```

Comment:
I would use indices instead of ids. In libMesh jargon, dof object ids are the ids of nodes/elements. Indices are used for indices in the DofMap

Reply:
Gotcha